doc/development.adoc

```diff
@@ -50,9 +50,14 @@ Build and install the Qt5 plugin http://pyotherside.readthedocs.io/en/latest/#bu
 
 ==== Linux (Fedora 29)
 
-    $ sudo dnf install qt5-devel qt5-qtbase-devel qt5-qtdeclarative-devel qt5-qtquickcontrols2-devel \
+    $ sudo dnf install make python \
+      libyubikey ykpers yubikey-manager \
+      qt5-devel qt5-qtbase-devel qt5-qtdeclarative-devel qt5-qtquickcontrols2-devel \
       qt5-qtquickcontrols qt5-qtgraphicaleffects pyotherside
 
+The `yubikey-manager` isn't strictly required to build `ykman-gui`, but
```

Comment:
I'd drop this comment, this is a dependency for a functional app. But the dependency is on the `python3-yubikey-manager` package, not on `yubikey-manager` which is the CLI tool.

Reply:
@dagheyman done - merge?